cmd/kritis/admission/main.go

```diff
@@ -65,8 +56,37 @@ func main() {
 		fmt.Println(version.Commit)
 		return
 	}
+
+	kritisConfigs, err := kritisconfig.KritisConfigs("")
+	if err != nil {
+		errMsg := fmt.Sprintf("error getting kritis config: %v", err)
+		glog.Errorf(errMsg)
+		return
+	}
+	if len(kritisConfigs) == 0 {
+		glog.Errorf("No KritisConfigs found in any namespace")
+		return
+	} else if len(kritisConfigs) > 1 {
+		glog.Errorf("More than 1 KritisConfig found, will use the 0th object")
+	}
+
+	kritisConfig := kritisConfigs[0]
+	// TODO(https://github.com/grafeas/kritis/issues/304): Use CRD validation instead
+	if len(kritisConfig.Spec.MetadataBackend) == 0 {
```

Comment:
since these are strings you check for empty string 
```if kritisConfig.Spec.MetadataBackend != "" {
} else {
}
```

Reply:
SG, PTAL.